### ISSUE TITLE
Request

### ISSUE THREAD

**Issue by cynical89** [BlueBubblesApp/BlueBubbles-Desktop-App#151]
for

>  Issue created by BlueBubbles Bot in the `BlueBubbles` commnity on behalf of `@CSab6482` in `#requests`.

**Comment by CSab6482** [comment]
Original title and description
Request for a macOS client

Sometimes, work or school Macs block Apple ID sign in, but we still want to be able to check our messages from the computer

**Comment by zlshames** [comment]
This should be as simple as just creating an additional build target for macOS.

**Comment by CSab6482** [comment]
Can this be closed? Or is it until 1.0.4 is an official release?

**Comment by zlshames** [comment]
> Can this be closed? Or is it until 1.0.4 is an official release?

Nah, not until it's in an official release, which it hasn't been yet
